api/config/cdr_versions_stable.json

```diff
@@ -9,5 +9,16 @@
     "numParticipants": 946237,
     "cdrDbName": "cdr20180831",
     "publicDbName": "public20180831"
+  },
+  {
+    "name": "Test Registered CDR 20180920",
```

Comment:
This name is a key. It's a better name than the old one .. But question is What happens when we switch the name ? Basically what you did is delete the cdr that was being used since friday in workbench and add this new one. 

@danqrodney What will happen to my workspace in test that i made with that old cdr on friday in workbench? 

We will find out. It is a good thing to test and see what happens. So i approve this name change .

Reply:
I believe it will fail with an error, here:

https://github.com/all-of-us/workbench/blob/master/api/tools/src/main/java/org/pmiops/workbench/tools/UpdateCdrVersions.java#L80

Which I guess will probably stop the deploy. Given that we know this is going to happen, perhaps we should not remove the old CDR version yet and change the code to be robust to this happening.

(Not deleting CDR versions with existing workspaces seems correct, but the tool should probably just log a warning and continue.)

Reply:
@danqrodney  Seems reasonable not to let deploy fail when we know its going to happen. So, we probably can let the test one be the same.